Version 4: clean up decorators so they are only for non-public attrib…

mattsmi opened the pull request:
Sorry!
I've had to correct a couple of things.
Please ensure you get all files from this request:

- input.txt
- output.txt
- README.md
- create_Constructor_Getters_Setters.py .

Some behaviour and spelling needed to be changed. Sorry for the bother!

Commit:
Version 4: clean up decorators so they are only for non-public attributes.

kaanturkmen commented:
Hey,
I have edited README.md after the merge. That's why it conflicted. Can you work with the new version? Thanks!

mattsmi commented:
> Hey,
> I have edited README.md after the merge. That's why it conflicted. Can you work with the new version? Thanks!

Sorry! Please check. After you apply my latest (last two) fixes, it should all be OK. I made the README.md clearer and corrected the English. Is that OK?

I also change input.txt and the script, and removed main.py. (That name is too generic and anyone using this script may easily overwrite it or lose it.)

Many thanks!

kaanturkmen commented:
Hello again,
I have merged your last 2 pr. I believe everything is fine now. Thanks a lot for contributions. Also let me know if anything is needed in this project. Take care!
